### ISSUE TITLE
Detection and segmentation performance

### ISSUE THREAD

**Issue by akuxcw** [Alibaba-MIIL/ImageNet21K#30]
I'm very interested in this awesome work! Do you try your pretrained model on other tasks like COCO detection or cityscapes segmentation? The performance on the classification tasks is amazing. I wonder whether it can improve other tasks.

**Comment by mrT23** [comment]
yes, we have results also on other tasks such as detection (MS-COCO) and retrieval (Inria holiday), that show consistent improvement when switching to 21K pretraining. we will add them to a future release of the paper.

you are welcome also to test yourself on your own downstream task, i am sure you will see improvement.

**Comment by akuxcw** [comment]
Hi, sorry to comment again. But I try resnet50_miil_21k.pth on semantic segmentation (cityscapes) using [mmseg](https://github.com/open-mmlab/mmsegmentation) with this [config](https://mycuhk-my.sharepoint.com/:u:/g/personal/1155113944_link_cuhk_edu_hk/ETFVyxAhYkVHgHJYGI8SUpgBAI7zqD3yITrUe1Hs12M-Wg?e=iZCpPR). And the result is much lower than traditional ImageNet-1k pretrained-weights (74.2 vs 76.5). Do you have any suggestions or try any semantic segmentation tasks?

**Comment by mrT23** [comment]
i have no experience with semantic segmentation or your package, but your results are a warning sign for you.
- are the results even close to SOTA  ? (my guess is no)
- are the results statistically stable?
- are the hyperparameters reasonable (learning rate, optimizer, ... )?
- search for bugs\problems (for example, are you properly loading the model)

my advice for you is to thoroughly check for bugs and inner problems. also move to a newer model (resnet50 is an old bad model )
